src/pages/Report.vue

```diff
@@ -2,7 +2,16 @@
   <div class="flex flex-col max-w-full">
     <PageHeader>
       <h1 slot="title" class="text-2xl font-bold">{{ report.title }}</h1>
-      <template slot="actions">
+      <template slot="actions">        
+        <Button
+          @click="link.action(reportData, filters)"
+          v-for="link of report.linkFields"
+          :key="link.label"
+          type="primary"
+          class="ml-2 text-white text-xs"
+        >
+          {{ link.label }}
+        </Button>
```

Comment:
This will cause all actions to be primary and overpopulate the _title bar_.

Export probably shouldn't be a primary action.

Also Export as JSON should be really be _Export_ drop down menu and _JSON_ as an item cause we'll be adding more filetypes `xlsx`, `pdf` and `csv`.

Items such as _Clear Filters_ can be under a different drop down menu such as _Actions_ or something.

Reply:
This makes sense and we will convert it into a drop-down like an option once we start adding multiple export options.

And I think GST report export JSON action should be a primary action here as that's what it will be used for.

Reply:
And have removed the hard coding of type primary and it will not take effect from the linkField values.

Reply:
Alright, I still find the blue _Export as JSON_ button too garish.

The issue is that the export button will be there (v0.0.5) on other doctypes and reports too, so this will break consistency as it won't be blue everywhere.

Anyways we can leave it in for now, I'll push the changes when taking care of exports.